CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# CONTRIBUTING GUIDELINES
+
+NovelRT is a community-driven project. Contribution is aslways welcome, and much appreciated.
+It is also how it'll be developed.
+
+First, please take a moment to review our [code of conduct](CODE_OF_CONDUCT.md).
```

Comment:
```suggestion
First, please take a moment to review our [code of conduct](CODE_OF_CONDUCT.md). We expect that all project participants will adhere to it, and any actions taken that go against the Code of Conduct will not be tolerated.
```

Good first line here! Just added a suggestion as we want to convey that actions against the CoC have zero tolerance in our eyes.